tests/host_scenarios/kubearmor-dev-next_test_1/cmd1

```diff
@@ -1,7 +1,7 @@
 source: kubearmor-dev-next
-cmd: screen -h
+cmd: diff -h
 result: failed
 ---
 operation: Process
-condition: screen
+condition: diff
 action: Block
```

Comment:
Can you please add a new line at the end of file?

Reply:
sure.